### PR TITLE
268_missing_number.py

### PR DESCRIPTION
**Pull request by Rwatana**
Please check my code

### CODE REVIEW

File: 268_missing_number.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from typing import List
+
+
+class Solution:
+    def missingNumber(self, nums: List[int]) -> int:
```
Comment (ryujumoto) [inline 268_missing_number.py:7]: This solution is wrong, so correct it to be accepted.

**Review by ryujumoto — COMMENTED** [review]
Check my review.

File: 268_missing_number.py (hunk 1 above)
Comment (Rwatana) [reply 268_missing_number.py:7]: I tried the different way. Please check

File: 268_missing_number.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from typing import List
```
Comment (ryujumoto) [inline 268_missing_number.py:3]: Write time complexity and space complexity.

**Review by ryujumoto — COMMENTED** [review]
Done.

**Comment by Rwatana** [comment]
I added them. Plase check my code

**Review by ryujumoto — APPROVED** [review]

**Pull request merged**